src/streaming/controllers/PlaybackController.js

```diff
@@ -268,20 +268,22 @@ MediaPlayer.dependencies.PlaybackController = function () {
             if (e.data.index === startIdx) {
                 firstAppended[id] = firstAppended[id] || {};
                 firstAppended[id][type] = true;
-                firstAppended.ready = !((stream.hasMedia("audio") && !firstAppended[id].audio) || (stream.hasMedia("video") && !firstAppended[id].video));
+                firstAppended[id].ready = !((stream.hasMedia("audio") && !firstAppended[id].audio) || (stream.hasMedia("video") && !firstAppended[id].video));
```

Comment:
@bbcrddave I will have @KozhinM address this line note.   This is part of his change.